package.json

```diff
@@ -19,11 +19,11 @@
     "simple-mock": "^0.4.1"
   },
   "dependencies": {
+    "fs-extra": "git+https://github.com/Rise-Vision/node-fs-extra",
```

Comment:
Patched version that requires our patched `graceful-fs` which imports `original-fs` in Electron